content/docs/ui/sending-email/single-sends.md

```diff
@@ -0,0 +1,137 @@
+
+## Work with Single Sends 
+
+### Create a Single Send
+
+_To create a new Single Send_
+
+1. From the left-hand navigation, select **Marketing**.
+2. Select **Single Sends** and click **Create Single Send**.
+3. Select a template from **Your Email Designs** or one of the pre-built **SendGrid Email Designs**.
+   <br> For more information about our designs, see [Working with Marketing Campaigns Email Designs]({{root_url}}/ui/sending-email/working-with-marketing-campaigns-email-designs/).
+
+<call-out>
+
+You can also create a template from an existing Single Send by selecting the action menu next to the email you'd like to reuse and selecting **Create Template**.
+
+</call-out>
+
+4. Select an editor. You can choose between the **Design Editor** and the **Code Editor**.
+   - Inside the Design Editor, the **Settings**, **Tags** and **Build** tabs allow you to design and configure your email.
+   - Inside the Code Editor, you can click the left **Settings** tab to expand the settings window, and then you can configure your email using the **Settings** and **Tags** tabs.
+
+![]({{root_url}}/img/design-editor-preview.png 'Design Editor Preview')
+
+5. Click **Save**. Your Single Send is now created. Next, you will need to edit the Single Send's settings.
+
+### Single Send required settings
+
+The **Settings** tab allows you to configure all the essential information about your Single Send, including the sender and recipients. To completely prepare a Single Send's settings, you must define the following fields.
+
+- **From Sender**: The publicly visible sender of your Single Send. This email address should be one of your verified senders. See our [sender verification documenation]({{root_url}}/ui/sending-email/sender-verification/) for more information about setting up a verified sender.
+- **Email Subject**: The email's subject line. This field can be A/B tested.
+- **Recipients**: The segments or lists you want to send the email to.
+
+<call-out type="warning">
+
+When selecting segments, make sure you are limiting your selection to 10 segments.
+
+See our [Segmentation documenation]({{root_url}}/ui/managing-contacts/segmenting-your-contacts/) for more information about managing your contacts and building Single Sends based on engagement data.
+
+</call-out>
+
+- **Scheduling**: A Single Send can be sent immediately or scheduled in advance. If you select **Specify a Date and Time**, **Date** and **Time** fields will be displayed where you can set a future send time.
+
+### Single Send optional settings
+
+In addition to the settings necessary to send your messages, Single Sends provide optional settings that allow you to more precisely manage how you target your audience with each message.
+
+#### Exclude contacts from a Single Send
+
+_To exclude a segment or list from your Single Send_
+
+1. Navigate to **Marketing** > **Single Sends**.
```

Comment:
I've tried to avoid the this > that > then this styling. I'd use "Navigate to **Marketing** and then select **Single Sends** or something like that